src/live_restore/live_restore_fs.c

```diff
@@ -1848,7 +1842,11 @@ __wt_os_live_restore_fs(
     F_SET(S2C(session), WT_CONN_LIVE_RESTORE_FS);
     if (0) {
 err:
-        if (lr_fs->os_file_system != NULL)
+        /*
+         * If we swapped in the posix file system don't terminate it. It'll get terminated later
```

Comment:
```suggestion
         * Live restore temporarily sets a posix file system so it can read configuration data from the turtle file before it initializes the live restore file system. If we error out while the posix file system is in use don't terminate it. It'll get terminated later
```

Reply:
This is all described above.

Reply:
[1d9dcf5](https://github.com/wiredtiger/wiredtiger/pull/11660/commits/1d9dcf5143d4fea8a82d3782989e7b9594842462)